docs/sql-ref-syntax-aux-describe-table.md

```diff
@@ -60,8 +61,95 @@ to return the metadata pertaining to a partition or column respectively.
+| ByteType              | `{ "type" : "tinyint" }`                                                                                              |
+| ShortType             | `{ "type" : "smallint" }`                                                                                             |
+| IntegerType           | `{ "type" : "int" }`                                                                                                  |
+| LongType              | `{ "type" : "bigint" }`                                                                                               |
+| FloatType             | `{ "type" : "float" }`                                                                                                |
+| DoubleType            | `{ "type" : "double" }`                                                                                               |
+| DecimalType           | `{ "type": "decimal", "precision": p, "scale": s }`                                                                   |
+| StringType            | `{ "type" : "string" }`                                                                                               |
+| VarCharType           | `{ "type" : "varchar", "length": n }`                                                                                 |
+| CharType              | `{ "type" : "char", "length": n }`                                                                                    |
+| BinaryType            | `{ "type" : "binary" }`                                                                                               |
+| BooleanType           | `{ "type" : "boolean" }`                                                                                              |
+| DateType              | `{ "type" : "date" }`                                                                                                 |
+| TimestampType         | `{ "type" : "timestamp_ltz" }`                                                                                        |
+| TimestampNTZType      | `{ "type" : "timestamp_ntz" }`                                                                                        |
+| YearMonthIntervalType | `{ "type" : "interval", "start_unit": "<start_unit>", "end_unit": "<end_unit>" }`                                     |
+| DayTimeIntervalType   | `{ "type" : "interval", "start_unit": "<start_unit>", "end_unit": "<end_unit>" }`                                     |
+| ArrayType             | `{ "type" : "array", "element_type": <type_json>, "contains_null": <boolean> }`                                       |
+| MapType               | `{ "type" : "map", "key_type": <type_json>, "value_type": <type_json>, "contains_null": <boolean> }`                  |
+| StructType            | `{ "name" : "<name>", "type" : "struct", "nullable": <boolean>, "comment": "<comment>", "default": "<default_val>" }` |
```

Comment:
How do we format the <default_val>? In some cases defaults can be expressions, if not today, then in the future. e.g. current_date(). So I propose to use an annotation that uses single quotes for strings (and date-time).

Reply:
for JSON parsing, the current implementation uses string double quoting for `default_val`

Reply:
The double quotes are because the default is a string.
But what goes into the string? So: It would be: "5" for a numeric, and "'hello'" for a string literal. "current_date()" for the current date expression.